packages/http-client-csharp/generator/Microsoft.Generator.CSharp/test/common/Helpers.cs

```diff
@@ -4,13 +4,14 @@
 using System;
 using System.Diagnostics;
 using System.IO;
+using System.Linq;
 using System.Runtime.CompilerServices;
 using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 
-namespace Microsoft.Generator.CSharp.Tests
+namespace Microsoft.Generator.CSharp.Tests.Common
 {
-    internal static class Helpers
+    public static class Helpers
     {
```

Comment:
We had 2 separate instances of this class in MGC and SCM plugin. I've consolidated this into one common helper that also enables us to test custom code in the SCM plugin.